### ISSUE TITLE
Every task should remember its last set proofing language

### ISSUE THREAD

**Issue by mkae** [dmfs/opentasks#308]
... for obvious reasons. :-)

**Comment by dmfs** [comment]
Do you mean the spell checker language?
Is there any way in Android for an app to set a custom spell checker language. In other words, can the system default be overridden temporarily by an app?

**Comment by mkae** [comment]
Yes, I mean that, but I have no idea whether it is possible to override the system's current setting.
